default/scripting/species/SP_NYMNMN.focs.txt

```diff
@@ -15,24 +15,23 @@ Species
     ]
 
     likes = [
-        "BLD_NEUTRONIUM_SYNTH"
-        "BLD_NEUTRONIUM_FORGE"
         "BLD_NEUTRONIUM_EXTRACTOR"
-        "POSITRONIUM_SPECIAL"
-        "MONOPOLE_SPECIAL"
+        "BLD_NEUTRONIUM_FORGE"
+        "BLD_NEUTRONIUM_SYNTH"
         "MINERALS_SPECIAL"
+        "MONOPOLE_SPECIAL"
+        "POSITRONIUM_SPECIAL"
     ]
     dislikes = [
-        "PLC_DESIGN_SIMPLICITY"
-        "FRACTAL_GEODES_SPECIAL"
-        "SPARK_FOSSILS_SPECIAL"
         "BLD_GAS_GIANT_GEN"
         "BLD_SOL_ORB_GEN"
-        "BLD_GAS_GIANT_GEN"
         "BLD_SPATIAL_DISTORT_GEN"
         "ECCENTRIC_ORBIT_SPECIAL"
         "FORTRESS_SPECIAL"
+        "FRACTAL_GEODES_SPECIAL"
         "GAIA_SPECIAL"
+        "PLC_DESIGN_SIMPLICITY"
+        "SPARK_FOSSILS_SPECIAL"
```

Comment:
this almost makes me want to propose to refactor all specials' names to begin with instead of end with SPECIAL ...

Reply:
i will not do that though. i could try and compress the committs somewhat, probably not all, or maybe redact at least some commit messages out if this here is getting close to finnish line

Reply:
also, i found no way to look up all possible focus types, only ways to look up foci for either a species or a planet, but not a place to find _all_ foci